### PR TITLE
separarte control over the sidekiq logging level

### PR DESCRIPTION
**Pull request by assafshomer**
@vinagrito @vitalis 
This is another approach (pushed by @vinagrito) to silent the excessive sidekiq `Start/Done` logs.
Instead of hacking just directly set the sidekiq facility log level.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ And then execute:
   ```
   defaults to: `Rails.application.class.parent.to_s.underscore`
 
+4. To separately control the Sidekiq logging level:
+
+  ```ruby
+  # config/environments/production.rb
+  MyApp::Application.configure do
+    config.applicaster_logger.sidekiq_level = Logger::ERROR # Logger::DEBUG etc..
```
Comment (vitalis) [inline README.md]: `sidekiq_level` -> `sidekiq_log_level`

**Comment by vitalis** [comment]
This PR / approach looks cleaner to me

File: README.md (hunk 1 above)
Comment (assafshomer) [inline README.md]: 👍

**Comment by vinagrito** [comment]
🙏

**Pull request merged**